pkg/antctl/raw/check/installation/test_denyall.go

```diff
@@ -84,6 +88,7 @@ func (t *DenyAllConnectivityTest) Run(ctx context.Context, testContext *testCont
 		testContext.Log("NetworkPolicy deletion was successful")
 		return nil
 	}()
+	time.Sleep(networkPolicyDelay)
```

Comment:
The *-deny-all e2e failures should have nothing to do with the changes for the stale resources cleanup, it's likely to be an issue with NP reconciliation which caused the service is connectable when deny is expected. So I added a 2 seconds to wait NP to be reconciled before the testing.

Reply:
I'd prefer something poll-based, but I guess this can be addressed in a later PR

Reply:
Sure, I can check later for poll-based with a new PR. Thanks.